package.json

```diff
@@ -1,29 +1,37 @@
 {
   "name": "json-rpc-middleware-stream",
   "version": "2.1.1",
-  "description": "",
+  "description": "A small toolset for streaming JSON RPC data and matching requests and responses.",
+  "license": "ISC",
+  "author": "MetaMask",
```

Comment:
Nit: Probably best to omit this. This is expected to be a person, and is parsed as such by `npm` for display on the registry website, and likely by various other tools as well.

```suggestion
```